legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabReviewApi.java

```diff
@@ -75,14 +75,19 @@ public class GitLabReviewApi extends GitLabApiWithFileAccess implements ReviewAp
 {
     private static final Logger LOGGER = LoggerFactory.getLogger(GitLabReviewApi.class);
 
+    public Predicate<String> getWorkspaceIdPredicate(String workspaceId)
+    {
+        return workspaceId == null ? null : workspaceId::equals;
+    }
```

Comment:
Look at EntityAccessResource for an example of how you might generate the predicate from the input string. Also, this should be moved to the resource class.